src/main/resources/templates/generators/overviewTrends.vm

```diff
@@ -5,7 +5,7 @@
   #includeTitle("Trends Overview")
 
   <script type="text/javascript">
-  #parse("/templates/js/trends-chart.js")
+  #parse("/templates/js/trends-chart.js.vm")
```

Comment:
I don't like this change because of:
- now some files are js, others vm
- by this change IDEA does not color syntax by default